docs/source/conf.py

```diff
@@ -48,7 +50,8 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
+# html_theme = 'alabaster'
+html_theme = 'default'
 html_theme_options = {
     'description': 'A Python library for the Podman service API',
     'fixed_sidebar': True,
```

Comment:
JFYI

WARNING: unsupported theme option 'description' given
WARNING: unsupported theme option 'fixed_sidebar' given

Reply:
Do you have a strong opinion on which theme is used?

Reply:
And can you merge please. TIA

Reply:
I really don't have an opinion at this time, just noting the warnings. having some docs are better than nothing. i'll merge this and we can play around with it later